### PR TITLE
 extended the README file by description of the eo-runtime release flow

### PR DESCRIPTION
**Pull request by Graur**
Closes: #1823

What's done:

- added short description  of the  `eo-runtime` release flow

### CODE REVIEW

**Comment by Graur** [comment]
@volodya-lombrozo Is it enough? Or you need some more details?

File: eo-runtime/README.md [hunk 1]
```diff
@@ -54,3 +54,17 @@ By default, dataization will be logged up until nesting level 3.
...
+
+Two things are needed to release eo-runtime:
+
+
+1. The first is the successful result of the `.rultor.yml` script in `objectionary/eo`.
+   Running this script will update the `gh-pages` git branch. Which contains only `.eo` objects from the `eo-runtime` package.
+   And this script will create several files in Maven central repository (eg `.jar`, `.pom` etc.).
+
+
+2. The second is to create a pull request in `objectionary/home` using the `pull.sh` script in a separate git branch.
```
Comment (volodya-lombrozo) [inline eo-runtime/README.md:68]: "in a separate git branch." - which one? Does the name of the branch matter?

Comment (volodya-lombrozo) [reply eo-runtime/README.md:68]: @Graur Should I run the next command in `objectionary/home`? 
```
./pull.sh objectionary/eo
```
If so, maybe it's better to provide step-by-step guide?

Comment (volodya-lombrozo) [reply eo-runtime/README.md:68]: Should I create PR by hands?

**Review by volodya-lombrozo — CHANGES_REQUESTED** [review]

File: eo-runtime/README.md (hunk 1 above)
Comment (Graur) [reply eo-runtime/README.md:68]: > "in a separate git branch." - which one? Does the name of the branch matter?

No, it doesn't

Comment (Graur) [reply eo-runtime/README.md:68]: > @Graur Should I run the next command in `objectionary/home`?
> 
> ```
> ./pull.sh objectionary/eo
> ```
> 
> If so, maybe it's better to provide step-by-step guide?

Sure, I will add more details

Comment (Graur) [reply eo-runtime/README.md:68]: > Should I create PR by hands?

Yes

**Review by volodya-lombrozo — CHANGES_REQUESTED** [review]
@Graur Please, have a look on [that PR](https://github.com/objectionary/home/pull/109). I've created it using your instructions. The problems which I faced with:
1. Script didn't removed old files (that were removed from `eo-runtime`). I had to do it [by hands](https://github.com/objectionary/home/pull/109/commits/11857a930791dca98aaf4d0efa138520ac38caa4).
2. I had to [update](https://github.com/objectionary/home/pull/109/commits/61482381bc8147f1818adc40871cdc8343f42e29) `eo.version` in `pom.xml` in `objectionary/home` in order to make it work.

Since it took me some time, I believe we also have to mention it in the docs. What do you think?

**Comment by Graur** [comment]
> @Graur Please, have a look on [that PR](https://github.com/objectionary/home/pull/109). I've created it using your instructions. The problems which I faced with:
> 
> 1. Script didn't removed old files (that were removed from `eo-runtime`). I had to do it [by hands](https://github.com/objectionary/home/pull/109/commits/11857a930791dca98aaf4d0efa138520ac38caa4).
> 2. I had to [update](https://github.com/objectionary/home/pull/109/commits/61482381bc8147f1818adc40871cdc8343f42e29) `eo.version` in `pom.xml` in `objectionary/home` in order to make it work.
> 
> Since it took me some time, I believe we also have to mention it in the docs. What do you think?

@volodya-lombrozo Sure, I agree. I will do it

**Comment by Graur** [comment]
@volodya-lombrozo What about now?

**Review by volodya-lombrozo — APPROVED** [review]
@Graur Thanks. Looks good to me.
@yegor256 Could you merge that changes please?

**Review by volodya-lombrozo — APPROVED** [review]
@Graur Thanks. Looks good to me.
@yegor256 Could you merge that changes please?

**Comment by Graur** [comment]
@volodya-lombrozo I fixed it, thanks!

**Comment by volodya-lombrozo** [comment]
@yegor256 Could you merge that changes, please?

**Comment by yegor256** [comment]
@Graur thanks!

**Comment by yegor256** [comment]
@Graur thanks!

**Pull request merged**